### DISCUSSION TITLE
node_modules/@fluentui/react/src/components/Button/Button.types.ts(2,28):

### DISCUSSION THREAD

**Discussion by jjrm1996** [microsoft/fluentui#21062] [Q&A] [answered]
Hello, I have the following error that depends on one of the modules of @fluentui/react. How can I fix it?

node_modules/@fluentui/react/src/components/Button/Button.types.ts(2,28):
![errorFluentUI](https://user-images.githubusercontent.com/33281059/146186637-a5ec0a9d-4c72-4622-9dfb-12fa62521b9b.PNG)
Thanks in advance.

**Comment by wonderplayer** [comment, marked as answer]
Hello,
How are you importing button to your component?
Looks strange, that the error is from `node_modules`.

**Reply by jjrm1996** [reply]
Sorry, I had misspelled a component within the import, I have been seduced your help, thank you.
